pkg/controllers/health/controller.go

```diff
@@ -0,0 +1,36 @@
+package health
+
+import (
+	"github.com/ViaQ/log-exploration-api/pkg/logs"
+	"github.com/ViaQ/log-exploration-api/pkg/middleware"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+type HealthController struct {
+	healthProvider logs.LogsProvider
+}
+
+func NewHealthController(router *gin.Engine, logsProvider logs.LogsProvider) {
+	healthController := &HealthController{
+		healthProvider: logsProvider,
+	}
+	r := router.Group("")
+	r.GET("/health", healthController.HealthHandler)
+	r.GET("/ready", healthController.ReadinessHandler)
+}
+
+func (healthController *HealthController) HealthHandler(gctx *gin.Context) {
+	middleware.AddHeader()
```

Comment:
@Yashwanth18  I think this middleware should be only used once when creating routes in the `NewHealthController` method. This will add it to the router, and then there is no need to call it explicitly.